include/seqan3/alphabet/aminoacid/aa27.hpp

```diff
@@ -31,28 +31,33 @@
 // DAMAGE.
 //
 // ============================================================================
-// Author: Sara Hetzel <sara.hetzel AT fu-berlin.de>
-// ============================================================================
+
+/*!\file aa27.hpp
+ * \ingroup aminoacid
+ * \author Sara Hetzel <sara.hetzel AT fu-berlin.de>
+ * \brief Contains seqan3::aa27, container aliases and string literals.
+ */
 
 #pragma once
 
 #include <cassert>
 
 #include <seqan3/alphabet/concept.hpp>
```

Comment:
no, I meant moving this down to the NDEBUG section as that is where the aklphabet concept is only used. OTOH the vector and string are always required so they need to stay up here!